CatwalkStation

EuSouAFazer opened the pull request:
## About The Pull Request

Adds CatwalkStation into the game.
Also adds its associated arrival shuttle, cargo shuttle and escape shuttle.
Also adds a few new floor variants: coldroom openspace and breathable air shuttle hulls.

Full Catwalk FAQ, Feedback Central and Why It's Good for the Game: https://hackmd.io/@kNRZ9uxhShS3s2N9scOXxQ/BJgfF0JhR

Also, I added a new tag for maps, so any Multi-Z map can opt out of having grappling hooks spawn in their bags. Catwalk was designed so the lower level is of easier access than the more restricted top access, so the problem of people falling into rooms they can't get out of is fixed, + it was giving a bunch of unintended access to places.

## Changelog
![365327162-97242637-002d-42d0-9882-aae73ebfa823](https://github.com/user-attachments/assets/c39068ee-9b41-43d4-b56b-dccfb122d8fc)
![365327135-201a800d-6d8b-471b-a73d-698d5d7e67ff](https://github.com/user-attachments/assets/cae0e7bd-7452-45b9-8d34-4228fc89c9c4)
![365327148-b1edf06e-9581-4165-98e7-b728c58a421b](https://github.com/user-attachments/assets/0666b40a-0108-4a80-949b-fb3315fe0472)
![364005220-8d5df862-01fd-4216-86ea-c90b75769e12](https://github.com/user-attachments/assets/ddbb3ab0-973c-44bb-9255-9e5620845239)
![365327156-2396267a-7a0f-4eb8-90c7-c7fa16926b65](https://github.com/user-attachments/assets/a8ef57e6-c779-4eb8-9796-2f63e65c0eb8)



:cl:
add: Adds CatwalkStation
code: Multi-Z maps now can toggle on or off having grappling hooks in people's bags
/:cl:

Ical92 committed:
AI Sat is finished pretty much !!

EuSouAFazer committed:
Shadow6's changes + new middle

EuSouAFazer committed:
Adds in shadow6's changes + Center Area

EuSouAFazer committed:
[Pushing for shadows6] Lights

EuSouAFazer committed:
shadows6 solar

Also features the abandoned chapel maint

Ical92 committed:
adds arrival shuttle / finishes up AI Sat

Ical92 committed:
adds arrival shuttle / finishes up AI Sat

New shuttle, wow !!
also finishes up some of the detailing on the AI Sat, done with it (for now)

Ical92 committed:
fixes shuttle oops misclick

Ical92 committed:
fixes shuttle oops misclick

oopsies forgot to update lights and also added a reinforced window somehow

Ical92 committed:
Fixes (almost) all Active Turfs

Ical92 committed:
Fixes (almost) all Active Turfs

woo yippee,, no more Danger! Low pressure and temperature detected.!!

EuSouAFazer committed:
Adds the Cargo Shuttle!

+ minor fixes on the Cargo Bay

EuSouAFazer committed:
Adds blob, carp and generic maint spawners

EuSouAFazer committed:
Adds generic event_spawn landmark

EuSouAFazer committed:
Miscelaneous Fixes

EuSouAFazer committed:
Chicken maint room

Ical92 committed:
Emergency Shuttle

Ical92 committed:
Emergency Shuttle

Adds emergency shuttle \o/
Original shuttle by shadow6

Ical92 committed:
Mo' Shuttles (cargone shuttled)

Ical92 committed:
Mo' Shuttles (cargone shuttled)

this shuttle took a significant part of my day and it was CentCom's fault

EuSouAFazer committed:
Improves solars + finished moth war room

Also removes a few dummy rooms floating around the station

Ical92 committed:
Fixes disconnected atmos devices

Ical92 committed:
Some duplicated pipes fixed

Ical92 committed:
Atmos Fixes Part One

Fixed all the unconnected devices
Made Virology disconnected from the main network
Fixed part of duplicated pipes

EEASAS committed:
Update CatwalkStation_2023.dmm

Ical92 committed:
Merge branch 'caming' of https://github.com/EEASAS/catwalkstation into catwalk

Ical92 committed:
Merge unstrider's edits

EEASAS committed:
rRAGGGFGDGSFDG

Ical92 committed:
Emergency Shuttle Changes / Fixes

fixes shuttle area(dont use area/shuttle for shuttle, its area/shuttle/escape)
other minor changes
replaces directional windows in eng with walls
replaces generic computers with terminals

Ical92 committed:
Small shuttle changes I forgot !!

Ical92 committed:
Small shuttle changes I forgot !!

Fixes some wallmounts / diagonal walls after looking in game, and placing some items that I missed

Ical92 committed:
Camera Fixes

Ical92 committed:
Camera Fixes

Fixes all mis-rotated and duplicate cameras

Ical92 committed:
No more var-edited cameras

Ical92 committed:
No more var-edited cameras

Changes all var-edited cameras to their directional subtypes
(Might not be necessary but good just incase)

Ical92 committed:
Job Spawns + Hangover Spawns

Ical92 committed:
More spawns and lots of stuff

Ical92 committed:
Jobs/Hangover Spawns + Door Accesses (not all)

Ical92 committed:
Southwest Maintenance work

Ical92 committed:
Southwest Maintenance work

See PR for image

EuSouAFazer committed:
Disposals

Ical92 committed:
Abandoned Medical

Ical92 committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation into catwalk

EEASAS committed:
Update CatwalkStation_2023.dmm

Ical92 committed:
Fix Cargo Mech and ATs

Ical92 committed:
EEASAS Edits

Ical92 committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation into catwalk

EEASAS committed:
Update CatwalkStation_2023.dmm

Ical92 committed:
ATs

Ical92 committed:
Removes HoS stamps and Hats from unsecure areas

Also give HoP a piano to drop on his conveyor, and adds some reinforced ceilings to command areas

Ical92 committed:
Resolve Updates

Ical92 committed:
Abandoned Medical, Cargo Mech Name, and ATs

Finishes up the abandoned medical hallway
Hopefully fixes the issue with the cargo mech's name growing every commit
Fixes some new active turfs

Ical92 committed:
Atmos Project/Turbine Room

Ical92 committed:
Cargo Conveyors / Shuttle, Railing Fix

Ical92 committed:
Fixes Stairs

EEASAS committed:
Update CatwalkStation_2023.dmm

EEASAS committed:
Update CatwalkStation_2023.dmm

EEASAS committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Merge pull request #46 from EEASAS/yrod

Update CatwalkStation_2023.dmm

Ical92 committed:
Ordnance Storage

Ical92 committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation into catwalk

Ical92 committed:
Atmos Project / Turbine Room + General Fixes

EEASAS committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
filled maints around engineering

Update CatwalkStation_2023.dmm

Ical92 committed:
Department Access

Ical92 committed:
Department Access

Added access helpers for all departments, maintenance access helpers still need to be done

Ical92 committed:
Firelocks (not all)

EuSouAFazer committed:
Fixes disposals

Ical92 committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation into catwalk

Ical92 committed:
Firelocks (not all)

Starting on putting in firelocks / alarms / air alarms
Mostly in Medical

Ical92 committed:
HFR Atmos Fixes & EVA Fix

Ical92 committed:
HFR Atmos Fixes & EVA Fix

Fixes the initial gas mix for the HFR Room and fixes the second floor of EVA to align with the lower ladder.

EEASAS committed:
Update CatwalkStation_2023.dmm

EEASAS committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Merge pull request #52 from EEASAS/yrod

Changes to security

Ical92 committed:
Science Airlock Names / Variations + Security Post Fixes

Ical92 committed:
Science Airlock Names / Variations + Security Post Fixes

Makes some airlocks in science glass airlocks, and fixes names
Standardized security post airlock names and fixed access

Ical92 committed:
Many things

Ical92 committed:
Missing APCs / General Fixes

Adds in APCs for areas that lacked them / removed duplicate APCs
Added /turf/open/openspace/coldroom to resolve ATs in the abandoned medical coldroom
Added APC + Atmos Devices for auxillary tool storage
Removes wires underwall in/around security
Flipped CMOs bed to face away from the wall (like any sane person would)
Slightly rearranges ordnance lab, fixes atmos piping, adds burn chamber air alarm

Ical92 committed:
Lights, Pods, Fixes

Ical92 committed:
Lights, Pods, Fixes

Adds lights and general fixes to some rooms
Changes to escape pods
Fixes AI powernet
Fixes some duplicate wires/piping

EuSouAFazer committed:
Changes some drunk spawn + top Z level of service/sec maint

EuSouAFazer committed:
Security Closet above Courtroom

EuSouAFazer committed:
Fixes a multi-Z problem in the minicity above viro

EEASAS committed:
Update CatwalkStation_2023.dmm

Rex9001 committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Merge pull request #56 from tritium-corporation/walimh

Update CatwalkStation_2023.dmm

Rex9001 committed:
Update CatwalkStation_2023.dmm

Rex9001 committed:
Merge remote-tracking branch 'upstream/master' into fixes_n_such

EuSouAFazer committed:
Fixes quite a few doubled up wires and pipes

EuSouAFazer committed:
Some more wire and pipe fixes

Rex9001 committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Merge pull request #57 from Rex9001code/fixes_n_such

Fills up maint south from the freezer

EEASAS committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Merge pull request #58 from tritium-corporation/schlattsoundsjustlikeariamathitscrazy

Update CatwalkStation_2023.dmm

Ical92 committed:
fixes,, so many fixes and also tests

EuSouAFazer committed:
Merge pull request #60 from Ical92/catwalk

fixes,, so many fixes and also tests

EuSouAFazer committed:
Adds APCs to rooms that didn't had them

EuSouAFazer committed:
Adds air alarms

EuSouAFazer committed:
Adds a maint room

EuSouAFazer committed:
Fixes brigs timers

EuSouAFazer committed:
Merge branch 'master' into master

EuSouAFazer committed:
Update emergency.dm

EuSouAFazer committed:
Number of fixes

EuSouAFazer committed:
More fixes

EuSouAFazer committed:
Fixes changes + adds new funny mine

haha. funny mine!

EuSouAFazer committed:
Decorates a little bit

EuSouAFazer committed:
Merge branch 'master' into master

EuSouAFazer committed:
Fixes stuff

EuSouAFazer committed:
Number of fixes

EuSouAFazer committed:
Adds missing maintenance accesses

EuSouAFazer committed:
Merge branch 'master' into master

EuSouAFazer committed:
Fixes broken typepaths

EuSouAFazer committed:
Fixes the supply shuttle

EuSouAFazer committed:
Fixes more problems - linters for uppercase? More likely than you'd think!

EuSouAFazer committed:
Changes the setup

EuSouAFazer committed:
also fixes the top z-level

EEASAS committed:
A BOOTLOAD TRUCKLOAD OF FIXES

EuSouAFazer committed:
Merge pull request #61 from tritium-corporation/master

A BOOTLOAD TRUCKLOAD OF FIXES

EuSouAFazer committed:
fixes windoors

GremlinSeeker committed:
Merge remote-tracking branch 'upstream/master' into pr/86442

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

linters check

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
linter check

linter

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

more lint check

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

one more check

GremlinSeeker committed:
lint check

check lint

GremlinSeeker committed:
last linters 🙏

GremlinSeeker committed:
🙏 should be all of them? 🙏

GremlinSeeker committed:
done?

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

compile map fix? merge commit

GremlinSeeker committed:
Merge remote-tracking branch 'upstream/master' into pr/86442

GremlinSeeker committed:
Merge branch 'tgstation:master' into master

GremlinSeeker committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

error check

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

test

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
vents

added scrub/air vent

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

test

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Adds a bunch of fixes

EuSouAFazer committed:
More maintenance work

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

airlock helpers for maints placed + unrestricted access side helpers, added fire extinguishers

EuSouAFazer committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation

EuSouAFazer committed:
Adds sec delinquent storage

EEASAS committed:
Update CatwalkStation_2023.dmm

EEASAS committed:
stuff

EEASAS committed:
Update CatwalkStation_2023.dmm

no more sandwich

EuSouAFazer committed:
Merge pull request #63 from tritium-corporation/gub

Update CatwalkStation_2023.dmm

EEASAS committed:
Stuff

EEASAS committed:
Darkandgrittyfication

EuSouAFazer committed:
Merge pull request #64 from tritium-corporation/gab

Stuff

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

cameras/linters fix

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

camera touch ups

EuSouAFazer committed:
Finishing maintenance touches

Masterpainisman commented:
oh god

EuSouAFazer committed:
Merge branch 'master' into master

EuSouAFazer committed:
Merge branch 'master' into master

EuSouAFazer committed:
Last minute fixes

Singul0 commented:
Going to drop an IED on an unsuspecting target while they walk down the corridors

thetaro commented:
everyone spawns with wirecutters, start spleefing security

deathrobotpunch commented:
This looks like an assassins creed map

Loafin34 commented:
What was that? Must've been the wind.

GremlinSeeker committed:
Merge branch 'master' into master

improvedname commented:
![image](https://github.com/user-attachments/assets/6013f739-8a6c-498f-9803-72bd507f3957)
![image](https://github.com/user-attachments/assets/e540fcd8-94f6-46f7-9c9d-1fdd1a1f2105)

Ordnance should use the proper areas for the burn chamber. Ordnance is also missing the freezing chamber and the mixer machine

![image](https://github.com/user-attachments/assets/31c2075e-fd45-499f-b7aa-00ff03c2a8d9)
![image](https://github.com/user-attachments/assets/b28d5482-f7cc-4574-8608-9cabbd120b14)

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

EEASAS committed:
adds a single billboard to the map

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

Deadgebert commented:
Few things I noticed.
Atmos is really, really, really bad. Not much space, no crystallizer, supplies are easy to mix, the big 3x3 gas tanks being right next to SM means any boom is unrecoverable, as you've vented all of the oxygen/n2. atmos mistakes shouldn't flood into engineering and the SM, new atmos players make a lot of mistakes the round shouldn't end because of it.
ordnance launch must be set to high to cross the floor outside of the launch chamber in order to have your TTV actually leave the station. you're going to get a lot of mistakes and bad faith players 'accidentally' bombing half the station with a maxcap.
you have a lot of broken stairs and missed wires across the station, sorry, i can't point them out, just boot it up and look for red APCs.
chemistry has a grinder on top of the chem dispenser on the top floor.

EuSouAFazer committed:
Removes hooks from catwalk

EEASAS committed:
fixes more checks

EEASAS committed:
some changes

EEASAS committed:
brings back command doors

EEASAS committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Merge pull request #65 from tritium-corporation/grah

adds a single billboard to the map

EEASAS committed:
meteor shields

EEASAS committed:
fixes a reagent grinder in chem

EEASAS committed:
Update CatwalkStation_2023.dmm

EEASAS committed:
i forgot to say but the previous commit adds shutters to chemistry lol sorry i love tgstation i love working for catwalk station i love mapping and this is really long wow i didnt know that commit names could get this long so how is your day i drank a monster energy today

EEASAS committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
Merge pull request #66 from tritium-corporation/muhg

meteor shields

EEASAS committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
fixes

Update CatwalkStation_2023.dmm

EEASAS committed:
fixes a singular pipe

EuSouAFazer committed:
Merge pull request #68 from tritium-corporation/fff

fixes a singular pipe

Maurukas reviewed (changes requested):
First pass review:

* Strongly recommend a pass for directional signs. In particular evac signs - there is only one directional evac sign on the entire station. Lots of places have only one sign. Be excessive, especially on a new station people haven't memorized.
* Spawner at 124,158,1 spawns over an invalid tile
* Security Locker room has no APC
* No Air Alarm in Aft Primary hallway, arrivals
* There are no lights in the kitchen itself, it seems to get some light from above
* There are 88 areas without light switches.
* There are 61 areas without intercomms.

<details><summary>Active Turfs at Aft Starboard Maintenance  154,73,2</summary>
<br>
Appears to be airless turfs used in an area open to maint.

![image](https://github.com/user-attachments/assets/660ff3b3-5c08-4835-941f-52c812692648)
</details>


<details>
<summary>Cosntruction storage has an APC,  no air alarm, but does have active atmos 
</summary>

![image](https://github.com/user-attachments/assets/1453f6e1-e98e-47f0-99df-e943058d5cdb)

</details>
<details>
<summary>Atmospherics/Engineering needs a significant amount of work. 
</summary>

* There are hidden pipes that need to be visible. There are loops and other problems: 

![image](https://github.com/user-attachments/assets/00d6ab5a-e817-4461-b5a9-eb3366718a1c)

* Atmos filter on the SM isn't defaulted to N2. Someone will screw this up.
* Supermatter engine needs an `air_sensor/engine_chamber` and associated helpers
* Engine emitters are not anchored/welded - ok if intentional
* The pumps are labeled but it is not obvious what each of these pipes do. Consider naming the gas flow meters as well: 

![image](https://github.com/user-attachments/assets/3b9bc579-2aa6-49a8-9954-392c8884d9f1)

* This is the most important gas pump on the station (Air to Distro) and the second most important gas pump on the station (Waste to Filter). These two pumps are usually given their own room. It should be **extremely obvious** where these two pumps are, and at the moment they kind of blend into the background, and are onon separate Z-levels from each other

![image](https://github.com/user-attachments/assets/3170bff5-8d8f-414b-8390-510d592ee5aa)
![image](https://github.com/user-attachments/assets/32034be1-40c3-4905-a985-aa23073090e8)

* Waste and filter are disconnected:

![image](https://github.com/user-attachments/assets/7631f6ca-54db-4e2c-97d3-bcd31b99f731)

* Engineering console is not on the station main grid, so AmpCheck won't work 

![image](https://github.com/user-attachments/assets/b5394f4d-4667-47a4-8c2f-f0070f06bacb)

</details>

<details>
<summary>
It is very not obvious if the cargo shuttle is docked from the upper platform
</summary>

Consider additional `status_display/supply` screens. Also you have to step onto the conveyor belt to reach the top lever: 

![image](https://github.com/user-attachments/assets/b856e120-5f78-4bff-810c-917258f5e34d)

</details>
<details>
<summary> Piping Issues:
</summary>

* There are a bunch of unit tests failing for doubled up pipes. They give specific coordinates so should be fairly easy to run through.
* 106,143 in Hydroponics:

![image](https://github.com/user-attachments/assets/d43d29c8-4425-49a4-9bbc-7f8b0f223c8a)

* 140,150,1 in primary hallway: 
![image](https://github.com/user-attachments/assets/fa1316fb-7dfc-475a-88d4-6966db64f325)

* One in the brig: 
![image](https://github.com/user-attachments/assets/234da956-3bd9-4143-82bc-2bb7cf3b7a3f)


</details>

Maurukas commented:
The big critical issue I see is the broken waste/filter loop connection. Get that patched and we're good to test merge.

EuSouAFazer committed:
Does what the doc (maptainer) ordered

EuSouAFazer committed:
oops a bit more changes

EuSouAFazer committed:
Adds signs

EuSouAFazer commented:
> First pass review:
> 
> * Strongly recommend a pass for directional signs. In particular evac signs - there is only one directional evac sign on the entire station. Lots of places have only one sign. Be excessive, especially on a new station people haven't memorized.
> * Spawner at 124,158,1 spawns over an invalid tile
> * Security Locker room has no APC
> * No Air Alarm in Aft Primary hallway, arrivals
> * There are no lights in the kitchen itself, it seems to get some light from above
> * There are 88 areas without light switches.
> * There are 61 areas without intercomms.
> 
> <details><summary>Active Turfs at Aft Starboard Maintenance  154,73,2</summary>
> <br>
> Appears to be airless turfs used in an area open to maint.
> 
> ![image](https://github.com/user-attachments/assets/660ff3b3-5c08-4835-941f-52c812692648)
> </details>
> 
> 
> <details>
> <summary>Cosntruction storage has an APC,  no air alarm, but does have active atmos 
> </summary>
> 
> ![image](https://github.com/user-attachments/assets/1453f6e1-e98e-47f0-99df-e943058d5cdb)
> 
> </details>
> <details>
> <summary>Atmospherics/Engineering needs a significant amount of work. 
> </summary>
> 
> * There are hidden pipes that need to be visible. There are loops and other problems: 
> 
> ![image](https://github.com/user-attachments/assets/00d6ab5a-e817-4461-b5a9-eb3366718a1c)
> 
> * Atmos filter on the SM isn't defaulted to N2. Someone will screw this up.
> * Supermatter engine needs an `air_sensor/engine_chamber` and associated helpers
> * Engine emitters are not anchored/welded - ok if intentional
> * The pumps are labeled but it is not obvious what each of these pipes do. Consider naming the gas flow meters as well: 
> 
> ![image](https://github.com/user-attachments/assets/3b9bc579-2aa6-49a8-9954-392c8884d9f1)
> 
> * This is the most important gas pump on the station (Air to Distro) and the second most important gas pump on the station (Waste to Filter). These two pumps are usually given their own room. It should be **extremely obvious** where these two pumps are, and at the moment they kind of blend into the background, and are onon separate Z-levels from each other
> 
> ![image](https://github.com/user-attachments/assets/3170bff5-8d8f-414b-8390-510d592ee5aa)
> ![image](https://github.com/user-attachments/assets/32034be1-40c3-4905-a985-aa23073090e8)
> 
> * Waste and filter are disconnected:
> 
> ![image](https://github.com/user-attachments/assets/7631f6ca-54db-4e2c-97d3-bcd31b99f731)
> 
> * Engineering console is not on the station main grid, so AmpCheck won't work 
> 
> ![image](https://github.com/user-attachments/assets/b5394f4d-4667-47a4-8c2f-f0070f06bacb)
> 
> </details>
> 
> <details>
> <summary>
> It is very not obvious if the cargo shuttle is docked from the upper platform
> </summary>
> 
> Consider additional `status_display/supply` screens. Also you have to step onto the conveyor belt to reach the top lever: 
> 
> ![image](https://github.com/user-attachments/assets/b856e120-5f78-4bff-810c-917258f5e34d)
> 
> </details>
> <details>
> <summary> Piping Issues:
> </summary>
> 
> * There are a bunch of unit tests failing for doubled up pipes. They give specific coordinates so should be fairly easy to run through.
> * 106,143 in Hydroponics:
> 
> ![image](https://github.com/user-attachments/assets/d43d29c8-4425-49a4-9bbc-7f8b0f223c8a)
> 
> * 140,150,1 in primary hallway: 
> ![image](https://github.com/user-attachments/assets/fa1316fb-7dfc-475a-88d4-6966db64f325)
> 
> * One in the brig: 
> ![image](https://github.com/user-attachments/assets/234da956-3bd9-4143-82bc-2bb7cf3b7a3f)
> 
> 
> </details>

Fixed everything, except the light switches and intercom which I'll go knocking off in the next few days. Other than that, let's get this testmerged >:]

EuSouAFazer committed:
Fixes a lot of stuff + HOPEFULLY tcomms is fixed

GremlinSeeker committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
More fixes

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

EuSouAFazer committed:
HoS' office refurbishing

EEASAS committed:
changes

connects permabrig to the powergrid, adds a security mech room with their paddy and replaces the elevator in the middle of sec with a safer one that doesnt gib people

EEASAS committed:
Update CatwalkStation_2023.dmm

fixes hos apc

EuSouAFazer committed:
Merge pull request #69 from tritium-corporation/something

Something

MrMelbert commented:
Camrah

![image](https://github.com/user-attachments/assets/b7efe78e-ab72-4043-bd3d-1b09c21c75df)

LSommers commented:
five "I AM ERROR" turfs above arrivals, southernmost edge

MrMelbert commented:
Mass driver in Ordnance should default to a higher power

MrMelbert commented:
Telecomms seems to depower itself rapidly

OuroborusReporter commented:
Genetics also lost power roughly 90 seconds into the round

BobDABuilder9001 commented:
![image](https://github.com/user-attachments/assets/2ace4d9e-f214-45af-964d-e7f68a255853)
Cool station so far!

Maurukas commented:
From latest round on Manuel:

* Access incorrect on 2nd floor Aux Surgery
* Morgue access is wrong (no paramedic or MD access)
* Only one Cryo Beaker
* Equipment on the floor in cryo - tables or racks for it?
* Weird visible -hidden pipe transition in medbay here 
![image](https://github.com/user-attachments/assets/f1536528-aa55-4574-92fb-3925f367cf4d)

* Multiple power grid issues medbay is cut off from grid at 119,99,1
* Medbay Central APC is covering a very large area - Consider breaking it up

EuSouAFazer committed:
More fixes

LSommers commented:
yes hi why is there a goddamn giant spider with a ton of HP compared to normal just randomly placed in service maints?

carshalash commented:
You used Ovens instead of ranges in the kitchen, chefs need ranges for soups and stuff. 

Also please for the love of god, expand the bar. That is way too small.

subject217 commented:
Not sure if these have been reported / fixed yet sorry

Brig entrance airlocks need to be linked to each other
"Navigate" feature does not seem to work for most locations
There are no security records consoles in the brig that are accessible to standard secoffs
Evac airlocks need to be linked to each other 

General thoughts on the brig - I think it's cool but it is a bit lacking in usable space in the main area. Especially if the elevator to permabrig is down for some reason - it's challenging to navigate with people moving around. Last round we had a cult and were trying to run deconversions in the brig main area and it was CRAMPED. I think the HoS office could do with being smaller if it meant more usable space for the brig as a whole.

SmArtKar reviewed (changes requested):
#### Some nitpicks

- Hangover landmark spam in holodeck will heavily skew the spawn weights and result in most of the crew spawning there.
- Engineering storage has a random roulette and prototype emitter - former is fine, but latter is a high powered manned turret that can be aimed and manually fired, so you may want to reconsider that choice (not saying that its a must)
#### Now much more importantly:
- Abandoned hallway doesn't have any markings, torn plating, etc - you may want to spice it up a bit
- Robotics and toxins feel extremely barren, not a fan of toxins being disjointed and in public access either, its a high security high danger area.
- Sci lab and public desk could benefit from being at least partially swapped places, protolathe and console being near windows/desks is an intentional choice on all maps. Not a fan of them being shoved into a tiny cupboard either (applies to engi as well)
- Main hallways feel like they severely lack decalwork, same applies to the entire southeastern corner of the first floor
- Engineering is in an extremely sad state and is most likely a blocker - merged atmospherics and SM chamber are a hard no, there's barely any room for projects. Multi-Z atmospherics is also a hard no, working with multi-level piping is agony and a catwalk ceiling prevents you from making proper burn chambers without tearing the flooring up. Avoid overusing catwalk floors there in general as they blend in with piping very easily, and also cover pipes which is ***not*** what you want in atmospherics where you're expected to modify the setup. Sorry, but this area needs a major redo.
- CE's, RD's, HoP's and CMO's offices can be "broken into" with 15 iron by building a ***stairs*** frame and finishing it with iron, ladders aren't the only way to climb levels. In 2/4 cases, this gives you instant access to antagonist objectives from their suit storage units without even having to break open the lockers. In other 2/4 cases, this can be done from a public access area.

Otherwise seems very good to me, I'd love to see this in-game.

improvedname commented:
**Permabrig Errors and Nitpicks**
![image](https://github.com/user-attachments/assets/7255dc5d-2eff-45ed-989c-e5cba20f23f0)

- Theres Wires in the walls
- The Apc in the cell merges with the toilet which is visually very weird the apc would be better in the main area since prisoner cells are sub areas
- The small janitor custodial closet is very nice but they're mussing a bucket so they have no way to fill the janicart
- Atmos alarms in every single prisoner cell is weird but not too bad
- The Cell/Brig lockers ID for Cells 1 and 2 don't match so they aren't linked to eachother.
![image](https://github.com/user-attachments/assets/71c03743-ca5b-41e2-965a-49122a631c81)
![image](https://github.com/user-attachments/assets/5bf7c5b0-e523-4ad0-aa91-7ce9bbc9ef45)

improvedname commented:
**Solars Nitpicks and Errors**
![image](https://github.com/user-attachments/assets/f885aefe-8112-4fcd-bd22-352b9037f507)

-At West solars theres cable ontop of the adapter, adapters are already cables of their own and don't need a cable ontop of it to function.

![image](https://github.com/user-attachments/assets/f40d93d7-cfd3-4d3d-b4eb-1840054848a4)
![image](https://github.com/user-attachments/assets/0afd6b6b-9198-4b50-90ad-8132f95180bf)


- East solars have the same issue with cable ontop of adapter for both floors

improvedname commented:
Theres 2 Rogue decal lines just outside of brig
![image](https://github.com/user-attachments/assets/2d0719fe-fc99-4085-9f57-b4a7bc19e08e)

improvedname commented:
![image](https://github.com/user-attachments/assets/210c880b-6441-453b-b4c8-0e8e563e2f6f)

- Firelocks are needed here on the upper side

EuSouAFazer committed:
loganuk's changes

EuSouAFazer commented:
> You used Ovens instead of ranges in the kitchen, chefs need ranges for soups and stuff.
Fixed

> [Mauruka's comment]
Fixed all, except the large area on med

> [subject217's comment
Fixed all, thank you! Those haven't been reported yet. Added a small area near the cells with chairs to buckle excess prisoners.
![image](https://github.com/user-attachments/assets/d2d9ea85-4bd1-4aba-9322-635e999759ac)

> [improvedname](https://github.com/improvedname)
Fixed them all, thanks for reporting

> [Smartkar's comment]
Fixy fixy fixy.

New engineering had a lot of changes - it still likely will need a full rework in the future, however this will at least make it playable during the testmerge. If it's good enough to be merged I don't know, but while we work on a more permanent solution we made it more playable.
![image](https://github.com/user-attachments/assets/f20c1b53-d233-42b0-976c-a0c1dd8a50de)

> yes hi why is there a goddamn giant spider with a ton of HP compared to normal just randomly placed in service maints?

Mr Johnson is very cuddly and is best avoided.

EuSouAFazer committed:
fuckton o fixes + engineering overhaul

EuSouAFazer committed:
CMO protection, almost forgot

EuSouAFazer committed:
last minute AI screens

Metekillot commented:
I know Smartkar briefly touched on it but to reiterate:


The current roundstart supermatter setup is extremely likely to delaminate, and the current roundstart atmospherics setup mixes the nitrogen, oxygen, and air supplies to where they are in fact just three chambers of 50/50 nitrogen and oxygen. Among other things, this means you can't pipe in new nitrogen for the SM without building your own filter arrangement to do so, and you can't refill oxygen canisters without doing something similar. Distro being a ratio of 50/50 also means fires will burn much, much hotter and for a longer time than they would otherwise.

The roundstart power monitoring console also only monitors one single room.

EuSouAFazer committed:
Final fixes to atmos

Chappyfart commented:
Maybe Im blind but where is secs FREE paddy...

Kurgis commented:
Science doesn't start with any data disks, and no computers have NT Frontier. Additionally, there's no ecto-sniffer machine in Robotics.

There's also no way for Robotics to leave their Mech Bay open without putting their workplace on all access, but that's less of an issue.

MrMelbert commented:
Think the kitchen coldroom's air alarm does not have the cold room preset?

MrMelbert commented:
Kitchen coldroom is not set correctly anyway, the freezer's connected to a vent with on air in it.

MrMelbert commented:
Also the map needs more navigation landmarks. It only has 5

MrMelbert commented:
Air vents are the old way to cool rooms. You should be using passive vents 
![image](https://github.com/user-attachments/assets/29511c9b-6c35-4dec-a6c4-a401257c33dd)

MrMelbert commented:
#90558 Was merged so you should do it too

SmArtKar committed:
Improves toxins on Catwalk

SmArtKar committed:
lighting and broken toxins pump

EuSouAFazer committed:
Merge pull request #70 from SmArtKar/catwalk_toxins

Improves toxins on Catwalk

EuSouAFazer committed:
Minor fixes

EuSouAFazer committed:
Merge branch 'master' into master

EuSouAFazer committed:
Fixes kitchens

EuSouAFazer commented:
> Maybe Im blind but where is secs FREE paddy...
Behind the gulag shuttle room

> Science doesn't start with any data disks, and no computers have NT Frontier.
Fixed

> [Melbert's issues]
Fixerino

OverwatchVoice commented:
As mentioned, station distribution starts with the air mixing going back into itself, resulting in a 1:1 mix that effectively bypasses the mixer. Offending pipe highlighted below
![obrazek](https://github.com/user-attachments/assets/f4d3caa8-e7ab-44c9-9dca-4c3069c996a8)

K4rlox commented:
paramedics dont have a dispatch area for some reason therefore the paramed suit is unobtainable

EuSouAFazer commented:
> As mentioned, station distribution starts with N2 and O2 pure networks linked to each other at multiple locations, effectively bypassing the mixer. Offending pipe connections highlighted below.
> ![obrazek](https://github.com/user-attachments/assets/43268ebb-2ad3-48ca-bcda-29992488bba9)
> 
> And here's with the connections unscrewed. 
> ![obrazek](https://github.com/user-attachments/assets/bcf861fb-a5ec-40cd-b561-0b0b05513ea6)
> 
> The N2 filter's filter output also connects back to the unfiltered pass.
> 
> Whole color coding of atmos could use a rework, too, it's all over the place.

Sorry about this - we had a big revision of Atmos done yesterday night but the code only got pushed to the live server today a few minutes ago, so a few rounds were played under the previous version. If/when y'all play with the new atmos do post here to keep me updated and ready for more changes


RE paramedics, added to do-do

EuSouAFazer committed:
Comamnd is no longer AA

EuSouAFazer committed:
FIX PIPES

EuSouAFazer committed:
ENGINEERING 2: Bogaloo's Strike

EuSouAFazer committed:
Cryo fix

Maurukas commented:
Roundstart ATs in N2 tank - got a standard tile in there: 
![image](https://github.com/user-attachments/assets/34ae4546-8b14-41ff-b376-5231681a44b4)

Maurukas commented:
Cryo is missing a CO2 filter and connection to the waste loop - odd I swear it had one earlier:
![image](https://github.com/user-attachments/assets/b29c924a-ccec-4011-a61f-4d99c8eb24f6)

Maurukas commented:
The AI reports that the telecomms server is too close to the AI chamber which results in very annoying constant noise.

Maurukas commented:
AI SMES is not fully charged at roundstart and wiring leads nowhere (should be `smes/full` subtype):
![image](https://github.com/user-attachments/assets/cc05141c-88b1-4ca4-a886-910cb8d7abd9)

LemonInTheDark commented:
openturfs in space on the upper z level are `/turf/open/openspace/airless` instead of `/turf/open/space/openspace`, which breaks lighting on turfs with the nearspace area, fucks up starlight, and will lead to weird atmos behavior

Aliceee2ch commented:
![image](https://github.com/user-attachments/assets/d10f9c3f-37c6-48a9-a5c5-f6882888f2b2)
decal issue in pharmacy

EuSouAFazer committed:
16/04 adds modsuit + more

EuSouAFazer committed:
Please do not break tcomms

HalfPunch commented:
There is no roundstart DNA disks no Catwalk

EuSouAFazer committed:
refixes comms

LSommers commented:
Messages appear to be being doubled in the message monitor for some reason.
Also there is a plasma room with no ceiling which ends predictably.

LSommers commented:
Also the bridge is (still) all-access

LSommers commented:
Floating tube light in departures, two tiles south of the reinforced table

GremlinSeeker committed:
lotta fixes

fixes big issues/ adds airlock to top-z to AI chamber, fixed cameras AI upload, fixed some spots where it was easy to break in, warden sightline improved. Added more fire extinguishers all around. Radiation Shutters added to the SM. Fixed cold rooms, added cool room air alarm helpers. Disk boxes added to gene, food processor gets a table, fixed mismatched airlocks on chemistry lab. fixed mismatched tiles in sec.

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

added holopads

DaCoolBoss commented:
There are broken items in Security's formalware crate. Couple of the suit items using the wrong path, I think they're supposed to be hos & warden formal jackets.

https://github.com/tgstation/tgstation/issues/90669#issue-3000193245

Maurukas commented:
Cryo CO2 filter is not connected to the waste loop: 
![image](https://github.com/user-attachments/assets/d6bfa406-689f-4afd-8665-459ae017e17f)

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

fixed sec clothes, fixed morgue, made existing materials more readily apparent.

GremlinSeeker commented:
> Cryo CO2 filter is not connected to the waste loop: ![image](https://private-user-images.githubusercontent.com/66576896/435429096-d6bfa406-689f-4afd-8665-459ae017e17f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDUxNTY5MjUsIm5iZiI6MTc0NTE1NjYyNSwicGF0aCI6Ii82NjU3Njg5Ni80MzU0MjkwOTYtZDZiZmE0MDYtNjg5Zi00YWZkLTg2NjUtNDU5YWUwMTdlMTdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDIwVDEzNDM0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4OWUyNjkzZmVjNzgzMjFjNDJmYzhhNTc2M2M3MGVkNTcwMDkyMDcyOWNhNjA2OTk5MTVmNjUzMmY3YjU4MzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7GO9yWFVNt708Qh8wrZYgft0znzJjK69HkZokyaGaxw)

forgot to mention this in my changes, but also fixed.

Cyberboss reviewed (changes requested):
The arrivals shuttle moves backwards

EuSouAFazer committed:
Reworks the Kitchen

EuSouAFazer committed:
Maintspicing 1

EuSouAFazer committed:
Intercoms + Maintening 2

EuSouAFazer committed:
Station Fixes

EuSouAFazer committed:
Shuttlefix + navbeacons

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

GremlinSeeker committed:
Merge branch 'master' of https://github.com/NoeSilent/catwalkstation

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

fixes

GremlinSeeker committed:
Update CatwalkStation_2023.dmm

fix?

Aliceee2ch commented:
space mold consumes
![image](https://github.com/user-attachments/assets/a4c1b6a9-bbb9-476e-8b05-1884142b0258)

GremlinSeeker committed:
arrival shuttles fix

fixed arrivals shuttles not being to dock

EuSouAFazer committed:
Do not husk anymore

EuSouAFazer committed:
fixes pipe vars

EuSouAFazer committed:
Final pipe fix

EuSouAFazer committed:
removes dust from wall

EuSouAFazer committed:
no decal maint spawners

EuSouAFazer committed:
please for the love of god fix wires

EuSouAFazer committed:
other minor wire fix

EuSouAFazer committed:
fixes more wires

EuSouAFazer committed:
Pushed up

EuSouAFazer committed:
Fixes ONE wire

EuSouAFazer committed:
fixes tripled wire

EuSouAFazer committed:
fixes more cable

EuSouAFazer committed:
another cablefix

Maurukas reviewed (approved):
Tests pass, nothing negative from the headmins or other maptainers, so here we go.
